hawkular-bus-common/src/main/java/org/hawkular/bus/common/Bus.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.bus.common;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.Resource;
+import javax.enterprise.context.ApplicationScoped;
+import javax.jms.ConnectionFactory;
+import javax.jms.Destination;
+import javax.jms.JMSContext;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+/**
+ * @author jsanda
+ */
+@ApplicationScoped
+public class Bus {
```

Comment:
I initially created it as part of some refactoring yet to be proposed/discussed. That refactoring effort though is largely centered around adopting JMS 2.0 APIs. The purpose of this PR though is only to move off of ActiveMQ and onto the default messaging subsystem in WF 10 (i.e, Artemis). On that basis, I am perfectly fine with removing the class.